src/SUMMARY.md

```diff
@@ -68,42 +101,17 @@
   - [Storing Books](exercises/day-1/book-library.md)
```

Comment:
re https://github.com/google/comprehensive-rust/pull/913#issuecomment-1673898090: This is the line that makes `mdbook` output a `day-1/book-library.html` file.

Reply:
Thanks, fixed!